trigger/patch.go

```diff
@@ -93,13 +95,13 @@ func (t *patchTriggers) patchOutcome(sub *event.Subscription) (*notification.Not
 		}
 		ps := target.ParentStatus
 
-		if ps != evergreen.PatchSucceeded && ps != evergreen.PatchFailed && ps != evergreen.PatchAllOutcomes {
+		if !evergreen.IsFinishedVersionStatus(ps) && ps != patchAllOutcomes {
 			return nil, nil
 		}
 
-		successOutcome := (ps == evergreen.PatchSucceeded) && (t.data.Status == evergreen.PatchSucceeded)
-		failureOutcome := (ps == evergreen.PatchFailed) && (t.data.Status == evergreen.PatchFailed)
-		anyOutcome := ps == evergreen.PatchAllOutcomes
+		successOutcome := evergreen.IsSuccessfulVersionStatus(ps) && evergreen.IsSuccessfulVersionStatus(t.data.Status)
+		failureOutcome := (ps == evergreen.VersionFailed) && (t.data.Status == evergreen.VersionFailed)
```

Comment:
nit: remove unnecessary parentheses since the order of operations precedence will evaluate it the same anyways.